eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -338,4 +338,30 @@
     <Field name="~(MAX_DURATION|NULL_TXN)"/>
     <Bug pattern="MS_SHOULD_BE_FINAL"/>
   </Match>
+
+  <!--  The return value is ignored for purpose-->
+  <Match>
+    <Class name="class com.microsoft.azure.servicebus.primitives.AsyncUtil"/>
+    <Method name="~(completeFuture|completeFutureExceptionally)"/>
+    <Bug pattern="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE"/>
+  </Match>
+
+  <!--  The casting is correct, it is beyond the SpotBugs's ability to determine the correctness -->
+  <Match>
+    <Class name="com.microsoft.azure.servicebus.primitives.RequestResponseUtils"/>
```

Comment:
[SpotBugs]

Unchecked/unconfirmed cast from com.microsoft.azure.servicebus.rules.RuleAction to com.microsoft.azure.servicebus.rules.SqlRuleAction of return value in com.microsoft.azure.servicebus.primitives.RequestResponseUtils.encodeRuleDescriptionToMap(RuleDescription)